api/http.go

```diff
@@ -164,3 +173,39 @@ func (c *HttpClient) Upgrade(ctx context.Context, name string, chart string, val
 
 	return result.Status, nil
 }
+
+func getModifyPath(cluster, namespace, releaseName string) (string, error) {
```

Comment:
prefer doing validations as part of flags.Valid() or within API func `upgrade/install`